client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-add-license-notification/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { useTranslate } from 'i18n-calypso';
+import { ReactElement, useEffect, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import Banner from 'calypso/components/banner';
+import { setPurchasedProduct } from 'calypso/state/jetpack-agency-dashboard/actions';
+import type { PurchasedProduct } from '../types';
+
+import './style.scss';
+
+export default function SiteAddLicenseNotification( {
+	purchasedLicense,
+}: {
+	purchasedLicense: PurchasedProduct;
+} ): ReactElement | null {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const selectedSite = purchasedLicense.selectedSite;
+	const selectedProduct = purchasedLicense.selectedProduct;
```

Comment:
Nitpicking here, so feel free to disregard the comment, but I think we could do:
```
const { selectedSite, selectedProduct } = purchasedLicense;
```